common/src/main/java/org/apache/celeborn/common/network/client/TransportResponseHandler.java

```diff
@@ -96,31 +96,29 @@ public TransportResponseHandler(TransportConf conf, Channel channel) {
               ThreadUtils.newDaemonThreadPoolScheduledExecutor(
                   "push-timeout-checker", conf.pushDataTimeoutCheckerThreads());
         }
-        if (checkFetchTimeout) {
-          if (fetchTimeoutChecker == null) {
-            fetchTimeoutChecker =
-                ThreadUtils.newDaemonThreadPoolScheduledExecutor(
-                    "fetch-timeout-checker", conf.fetchDataTimeoutCheckerThreads());
-          }
-        }
+
+        pushCheckerScheduleFuture =
```

Comment:
IMO we don't need to put these lines into `synchronized (TransportResponseHandler.class)` block because `pushTimeoutChecker` is not static.